### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by laurapigeon** [mov-cli/mov-cli#365]
**Describe the bug:** When selecting the plugin namespace via the integrated selector (by not passing `-s`), if the plugin has only a default scraper, the user will be kicked out as if no scraper was chosen. The error will suggest setting a default scraper in `config.toml`, but this stops the integrated plugin selector from displaying.

**Mov-cli version:** ``4.4.15``

**Expected behaviour:** The default scraper should be used in the case where the selected plugin only has a default, like what happens when just passing the namespace in to `-s`.

**Steps to reproduce:** 

1. Install the `mov-cli-files` plugin (as an example with only a default scraper), and add it to the plugins config with namespace `foofiles`.
2. Invoke `mov-cli test` to enter the "Select a plugin:" selector, and choose the `foofiles` namespace you set up.
3. An error is thrown: `You must choose a scraper to scrape with! You can set a default scraper with the default key in config.toml.`
4. Instead invoke `mov-cli -s foofiles test` to select the namespace without selecting the scraper.
5. No error is thrown and the default scraper is used as expected.

**Screenshots:**

**Logs:**
```python
$ mov-cli --debug test
[DEBUG] (mov_cli) - Config -> {'version': 1, 'debug': True, 'player': 'mpv', 'quality': 'fhd', 'skip_update_checker': True, 'auto_try_next_scraper': True, 'hide_ip': True, 'ui': {'preview': True, 'watch_options': True, 'display_quality': False}, 'plugins': {'yt': 'mov-cli-youtube', 'foofiles': 'mov-cli-files'}, 'scrapers': {}, 'http': {'timeout': 15}}

Good Morning, laura.
    It's 10:43 AM on a magnificent Monday! 

[DEBUG] (mov_cli) - Scraper args picked up on --> []
[ERROR] (mov_cli) - You must choose a scraper to scrape with! You can set a default scraper with the default key in config.toml.
```
```python
$ mov-cli --debug -s foofiles test
[DEBUG] (mov_cli) - Config -> {'version': 1, 'debug': True, 'player': 'mpv', 'quality': 'fhd', 'skip_update_checker': True, 'auto_try_next_scraper': True, 'hide_ip': True, 'ui': {'preview': True, 'watch_options': True, 'display_quality': False}, 'plugins': {'yt': 'mov-cli-youtube', 'foofiles': 'mov-cli-files'}, 'scrapers': {'default': 'foofiles'}, 'http': {'timeout': 15}}

Good Morning, laura.
    It's 10:47 AM on a wonderful Monday! 

[DEBUG] (mov_cli) - Scraper args picked up on --> []
[INFO] (mov_cli) - Using 'foofiles.DEFAULT' scraper...
[INFO] (mov_cli) - Searching for 'test'...
```

**Platform:**

 - OS: Linux
 - Version: Arch (AUR)

**Checklist**
- [x] I have added logs.
- [x] I have added my platform.
- [x] I have added my version number.
- [x] I am sure that this issue is not related to a plugin.

**Comment by THEGOLDENPRO** [comment]
Yeah that doesn't seem right. In the meantime I think disabling `auto_try_next_scraper` in the config could temporarily fix this.

I'll try and look into this in my spare time, thanks for bringing up the issue with such detail too. 👍

**Comment by fractuna** [comment]
Hello
I tried to find the problem and I think I got some ideas.

If you see the cli/scraper.py, there is a function called 'propt' that choose scraper based on user input, but when you use a *plugin* that don't have any _scraper_ except 'DEFAULT', the 'plugin.scrapers' length will be equal to zero. Why? it's because plugin.scraper function act as a getter property and if you read the source of that property, you can see it ignores any scraper name ends with 'DEFAULT', and that's why the mov-cli wants us to provide the default scraper manually.

There is multiple way to fix this problem, but easiest way is to check the length of plugin.scrapers property and if there is not any other scrapers, we can return the default scraper.

Or we can just return the default scraper when prompt returns None value.

Also you can fix this bug with adding another scraper with arbitrary name but with the *same* class as DEFAULT scraper in your plugin __init__.py file.

I tried both solutions and they worked for me, but idk if those are good ways for fixing that.
